src/views/Simulation/utils/SceneContainer.js

```diff
@@ -96,6 +97,49 @@ export class SceneContainer extends Container {
     this.sceneApp.ticker.add(this.updatePositionAndScale);
   }
 
+  findElementById(elementId) {
+    const recurseSearch = (elements) => {
+      if (!Array.isArray(elements) || elements.length === 0) return null;
+      let elementFound = elements.find((child) => child.elementId === elementId);
+      if (elementFound != null) return elementFound;
+
+      for (const child of elements) {
+        elementFound = recurseSearch(child.children);
+        if (elementFound != null) return elementFound;
+        else continue;
+      }
+    };
+
+    return recurseSearch(this.children);
+  }
+
+  getScreenCenterPoint() {
+    return new Point(this.canvasScene.clientWidth / 2, this.canvasScene.clientHeight / 2);
+  }
+
+  getPointWithZoomOffset(x, y, zoom = this.zoom) {
+    return new Point(x * zoom, y * zoom);
+  }
+
+  centerOnElement(elementId) {
+    const elementFound = this.findElementById(elementId);
```

Comment:
Not a huge fan of having to search for an item in the containers to find its coordinates, it would be better to retrieve the element coordinates (computed by the layout) library, and simply translate them to the scene coordinates system. 
But I guess we can keep it like this for now. Performance shouldn't be a problem because this event is only triggered manually.

Reply:
The element coordinates computed by the layout offer no guaranty to got the final coordinates elements after transformations and scales. Moreover, the graphics links need some computes to get an element center and I think it is quite hard to get it only from the layout coordinates.